### DISCUSSION TITLE
Request - Serial to TCP Bridge parity support (ESP32)

### DISCUSSION THREAD

**Discussion by helmutacp** [arendst/Tasmota#19404] [Ideas]
Hi There!

I'm using Serial to TCP to connect serial devices by TCP. Unfortunately the device uses even parity.

Back at TasmotaSerial-3.3.0 I was able to modify the source myselfe ( "config = SERIAL_8E1").
TasmotaSerial-3.6.0 is beyond my programming skills to be able to customize it myself.

So I politly ask to add SerialConfig (set partity) to the Serial to TCP Bridge
or assist me with a command to change TasmotaSerial-3.6.0 myself (don't need dynamic customization facility).

Thanks in advance.

**Comment by sfromis** [comment]
Not using this, but I'd try the command `TCPConfig 8E1`

**Comment by helmutacp** [comment]
You are my hero sfromis!
Thanks, I looked at "Serial to TCP Bridge" but not at the "WiFi" section.
